templates/common/_base/files/etc-networkmanager-dispatcher.d-90-long-hostname.yaml

```diff
@@ -15,7 +15,8 @@ contents:
         exit 0
     fi
 
-    if [[ ! "$(< /proc/sys/kernel/hostname)" =~ (localhost|localhost.localdomain) ]]; then
+    kn="$(< /proc/sys/kernel/hostname)"
+    if [[ ! "${kn}" =~ (localhost|localhost.localdomain) ]] && [ "${kn}" -le 63 ]; then
```

Comment:
I think bash requires that you're comparing an int when doing the `-le` check, so you want `${#kn}` not the actual variable here I believe.

Reply:
Good catch....the `#` was intended.